ragger_tests/application_client/curve.py

```diff
@@ -0,0 +1,131 @@
+from abc import ABC, abstractmethod
+from typing import TypeVar
+
+from application_client.instruction_type import InsType
+from response_unpacker import PK, Curve25519PublicKey, SignedPayloadEd25519, SignedPayloadSecp256k1, Signed, Secp256k1PublicKey
+
+class Curve(ABC):
```

Comment:
An abstract class used by `SECP256K1` and `Curve25519` below, needed to produce a "generic type" below: `C = TypeVar('C', bound=Curve)`